src/internal/signalr/handler.go

```diff
@@ -80,6 +85,33 @@ func (o *observationsHandler) HandleObservation(observation Observation) error {
 	return nil
 }
 
+func (o *observationsHandler) handlePhaseMode(observation Observation) error {
```

Comment:
Let's sync on that, 'cause I think we need to extend it (not this method, but the overall functionality).